server/Controllers/GameControllers/snake.js

```diff
@@ -0,0 +1,16 @@
+class Snake {
+  constructor(playerId, initialPositionX, initialPositionY,
+    initialDirection, snakeColor, initialSnakeLength) {
+    this.ownerId = playerId;
+    this.headX = initialPositionX;
+    this.headY = initialPositionY;
+    this.direction = initialDirection;
+    this.color = snakeColor;
+    this.bodyLength = initialSnakeLength;
+    this.canvasCoordinates = [];
```

Comment:
what does `this.canvasCoordinates` represent?

Reply:
It's not a coherent name. will change it? It is the array of coordinates of the snake body.